src/ontodia/widgets/toolbar.tsx

```diff
@@ -73,6 +77,33 @@ export class EditorToolbar extends React.Component<Props, void> {
         );
     }
 
+    private renderButtonsTogglePanels = () => {
+        const {
+            hidePanels,
+            isLeftPanelOpen,
+            onLeftPanelToggle,
+            isRightPanelOpen,
+            onRightPanelToggle
+        } = this.props;
+
+        if (hidePanels) { return null; }
+
+        const className = `ontodia-btn ontodia-btn-default ${CLASS_NAME}__toggle`;
+
+        return (
+            <div className='ontodia-btn-group ontodia-btn-group-sm'>
+                <button type='button'
+                        className={`${className} ${CLASS_NAME}__toggle-left ${isLeftPanelOpen ? 'active' : ''}`}
```

Comment:
Please don't use non-BEM class names, e,g, `.active`. In this case, for example, it may look like this:
```
${CLASS_NAME}__toggle-left--active
```